ocaml/xapi/xapi_xenops.ml

```diff
@@ -593,11 +593,11 @@ module MD = struct
             RealTime (qos_class params)
         | "idle" ->
             Idle
-        | "best-effort" ->
+        | "best_effort" ->
```

Comment:
Too many things being replaced now...

Reply:
ah we already have 'best-effort' elsewhere

Reply:
I'd just update the datamodel and let the compiler tell me what else to update.